streamz/sources.py

```diff
@@ -467,6 +468,30 @@ def __init__(self, topic, consumer_params, poll_interval='1s',
     def poll_kafka(self):
         import confluent_kafka as ck
 
+        def commit(_part):
```

Comment:
Can these functions be moved outside this def? Dynamic function closures have a reputation for holding references.

Reply:
I'll have to re-import confluent_kafka every time the function is called